opencti-platform/opencti-front/src/private/components/settings/SessionsList.jsx

```diff
@@ -76,187 +74,167 @@ const styles = (theme) => ({
       onError: () => {
-        this.setState({ killing: false });
+        setKilling(false);
       },
     });
-  }
+  };
 
-  render() {
-    const { classes, nsdt, t, data, keyword } = this.props;
-    const sortByNameCaseInsensitive = R.sortBy(
-      R.compose(R.toLower, R.path(['user', 'name'])),
-    );
-    const filterByKeyword = (n) => keyword === ''
-      || n.user.name.toLowerCase().indexOf(keyword.toLowerCase()) !== -1;
-    const sessions = R.pipe(
-      R.propOr([], 'sessions'),
-      R.filter(filterByKeyword),
-      sortByNameCaseInsensitive,
-    )(data);
-    return (
-      <>
-        <List
-          component="nav"
-          aria-labelledby="nested-list-subheader"
-          className={classes.root}
-        >
-          {sessions.map((session) => {
-            const { user, sessions: userSessions } = session;
-            const orderedSessions = R.sort(
-              (a, b) => timestamp(a.created) - timestamp(b.created),
-              userSessions,
-            );
-            return (
-              <div key={session.user.id}>
-                <ListItem
-                  classes={{ root: classes.item }}
-                  divider={true}
-                  button={true}
-                  component={Link}
-                  to={`/dashboard/settings/accesses/users/${user.id}`}
-                >
-                  <ListItemIcon>
-                    <ItemIcon type="User" />
-                  </ListItemIcon>
-                  <ListItemText
-                    primary={
-                      <div>
-                        <div className={classes.name}>{user.name}</div>
-                        <div className={classes.email}>{user.email}</div>
-                      </div>
-                    }
-                  />
-                  <ListItemIcon classes={{ root: classes.goIcon }}>
-                    &nbsp;
-                  </ListItemIcon>
-                </ListItem>
-                <List style={{ margin: 0, padding: 0 }}>
-                  {orderedSessions.map((userSession) => (
-                    <ListItem
-                      key={userSession.id}
-                      classes={{ root: classes.itemNested }}
-                      divider={true}
-                    >
-                      <ListItemIcon>
-                        <ItemIcon type="Session" />
-                      </ListItemIcon>
-                      <ListItemText
-                        primary={
-                          <div>
-                            <div className={classes.created}>
-                              {nsdt(userSession.created)}
-                            </div>
-                            <div className={classes.ttl}>
-                              {Math.round(userSession.ttl / 60)}{' '}
-                              {t('minutes left')} /{' '}
-                              {Math.round(userSession.originalMaxAge / 60)}
-                            </div>
+  const sortByNameCaseInsensitive = R.sortBy(
```

Comment:
If it's not adding to much work (or code) it will be amazing to remove Ramda and use pure JS in this component :)

Reply:
Done :)